README.md

```diff
@@ -39,6 +40,8 @@ cronofyClient.requestAccessToken(options, function(err, response){
 
 The cronofy client object can be initialized with client and token details. These details will be automatically added to each call that they are needed for, but these values can be replaced by any specified in the method's `options` object.
 
+`data_center` is the two-letter designation for the data center you want to operate against - for example `de` for Germany, or `au` for Australia. When omitted, this defaults to the US data center.
```

Comment:
Worth mentioning where this comes from on the dashboard?

Reply:
Added a line for this.